### DISCUSSION TITLE
Copilot autocomplete is not working - VSCode

### DISCUSSION THREAD

**Discussion by shawonibnkamal** [community/community#117028] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I cant seem to find any relevant discussion on it. How can I troubleshoot this autocomplete not working for me in VSCode?

**Comment by Ramhubi** [comment]
say please its working😝

**Comment by hanson-wen** [comment]
me too, sometime ok, sometimes not working.

**Comment by vad-fedorov-datsteam** [comment]
<img width="372" alt="Снимок экрана 2024-04-09 в 19 56 42" src="https://github.com/community/community/assets/157374474/f108f440-827c-445c-b5de-955266a4dd8e">

### Code suggestions in VS Code does not work

At the same time, in other IDE, for example, **Webstorm**, code suggestions works. But, as it turned out, not everyone...

_P.S. the chat with the copilot works_

<details>
  <summary>Diagnostics from Github Copilot extension</summary>

## Copilot

- Version: 1.178.0
- Build: prod
- Editor: vscode/1.88.0

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Feature Flags

- Send Restricted Telemetry: disabled
- Chat: enabled

## Node setup

- Number of root certificates: 141
- Operating system: Darwin
- Operating system version: 22.5.0
- Operating system architecture: arm64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Network Configuration

- Proxy host: n/a
- Proxy port: n/a
- Kerberos SPN: n/a
- Reject unauthorized: disabled
- Fetcher: HelixFetcher

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- api.githubcopilot.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 

## VS Code Configuration

- HTTP proxy: 
- HTTP proxy authentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Extensions

- Is `win-ca` installed?: false
- Is `mac-ca` installed?: false

## Authentication

- GitHub username: vad-fedorov-datsteam

## Output Github Copilot
```
2024-04-09 19:51:04.341 [info] [fetcher] Using Helix fetcher.
2024-04-09 19:51:04.341 [info] [code-referencing] Public code references are enabled.
2024-04-09 19:51:04.341 [info] [auth] Sucessfully authenticated
2024-04-09 19:56:32.457 [info] [fetchCompletions] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 273 ms
2024-04-09 19:56:55.046 [info] [fetcher] Using Helix fetcher.
2024-04-09 19:56:55.046 [info] [code-referencing] Public code references are enabled.
2024-04-09 19:56:55.309 [info] [fetchCompletions] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 87 ms
2024-04-09 19:56:59.105 [info] [fetcher] Using Helix fetcher.
2024-04-09 19:56:59.105 [info] [code-referencing] Public code references are enabled.
2024-04-09 19:56:59.284 [info] [fetchCompletions] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 82 ms
```
</details>

**Reply by ALKALI-CMD** [reply]
I have the same problem.

**Comment by balusena** [comment]
To troubleshoot Copilot autocomplete in VSCode, try reinstalling the extension, ensuring compatibility with your VSCode version. Restart VSCode and check for conflicting extensions. If the issue persists, review VSCode and Copilot documentation or seek support from the GitHub community.

**Reply by vad-fedorov-datsteam** [reply]
Tried everything, nothing helps:
- reinstalling the plugin
- installation of other versions of the plugin
- restarts
- etc

All the logs are adequate, the diagnostics pass, the chat works, but the code suggestions doesn’t work... And it worked about a month ago

**Comment by jfullstackdev** [comment]
it's based on interaction, sometimes it will not auto-complete, but the more intuitive one to be used is the Copilot Chat version, simply install the VSCode extension of it, you can highlight block of code that say you want to ask or find the bug, you can even sent it together with your questions, and it's more accurate doing the job, like say even a documentation, but it cannot read the entire codebase, you as the human should always navigate and correctly use the tool as it is intended

**Reply by vad-fedorov-datsteam** [reply]
We have problems not with understanding how to use the tool, but with the fact that the tool does not fulfill its declared functions, although money has been paid.

**Comment by bhoodream** [comment]
VSCode, version of the copilot plugin is **v1.194.0**, code suggestions are working

**Reply by rogaha** [reply]
any updates on this?

**Comment by alfianrehanusa** [comment]
its not working for me in newest version. Works perfectly yesterday
